src/mono/wasm/Wasm.Build.Tests/BuildPublishTests.cs

```diff
@@ -15,17 +15,23 @@ namespace Wasm.Build.Tests
 {
     public class BuildPublishTests : NativeRebuildTestsBase
     {
+        private readonly char unicodeChar = '煉';
+
         public BuildPublishTests(ITestOutputHelper output, SharedBuildPerTestClassFixture buildContext)
             : base(output, buildContext)
         {
         }
 
         [Theory]
-        [BuildAndRun(host: RunHost.Chrome, aot: false, config: "Release")]
-        [BuildAndRun(host: RunHost.Chrome, aot: false, config: "Debug")]
-        public void BuildThenPublishNoAOT(BuildArgs buildArgs, RunHost host, string id)
+        [BuildAndRun(host: RunHost.Chrome, aot: false, config: "Release", parameters: new object[] { false })]
```

Comment:
`BuildAndRun` supports extra attributes, for example:
https://github.com/radical/runtime/blob/519608467eb276c573ca191cf0f53ca983314247/src/mono/wasm/Wasm.Build.Tests/PInvokeTableGeneratorTests.cs#L519-L520

Reply:
I thought that that's exactly the mechanism I used :)

Reply:
lol sorry, I misread!!